docs/getting_started.md

```diff
@@ -0,0 +1,262 @@
+metadata:
+  name: foobar-controlplane
+  labels:
+    set: master
+    cluster.k8s.io/cluster-name: foobar
+spec:
+  providerSpec:
+    value:
+      apiVersion: "vsphereproviderconfig/v1alpha1"
+      kind: "VsphereMachineProviderConfig"
+      machineSpec:
+        datacenter: "SDDC-Datacenter"
+        datastore: "DefaultDatastore"
+        resourcePool: "*/Resources/Compute-ResourcePool/CAPV"
+        vmFolder: "Workloads"
+        networks:
+        - networkName: "vm-network-1"
+          ipConfig:
+            networkType: "dhcp"
+        numCPUs: 2
+        memoryMB: 2048
+        template: "ubuntu-16.04-server-cloudimg-amd64"
+        disks:
+        - diskLabel: "Hard disk 1"
+          diskSizeGB: 20
+        preloaded: false
+  versions:
+    kubelet: 1.13.6
+    controlPlane: 1.13.6
+```
+
+To add 3 additional worker nodes to your cluster, create a machineset like the following:
+
+```yaml
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineSet
+metadata:
+  name: foobar-workers
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      node-type: worker-node
+      cluster.k8s.io/cluster-name: foobar
+  template:
+    metadata:
+      labels:
+        node-type: worker-node
+        cluster.k8s.io/cluster-name: foobar
+    spec:
+      providerSpec:
+        value:
+          apiVersion: "vsphereproviderconfig/v1alpha1"
+          kind: "VsphereMachineProviderConfig"
+          machineSpec:
+            datacenter: "SDDC-Datacenter"
+            datastore: "DefaultDatastore"
+            resourcePool: "*/Resources/Compute-ResourcePool/CAPV"
+            vmFolder: "Workloads"
+            networks:
+            - networkName: "vm-network-1"
+              ipConfig:
+                networkType: "dhcp"
+            numCPUs: 2
+            memoryMB: 2048
+            template: "ubuntu-16.04-server-cloudimg-amd64"
+            disks:
+            - diskLabel: "Hard disk 1"
+              diskSizeGB: 20
+            preloaded: false
+      versions:
+        kubelet: 1.13.6
+```
+
+`kubectl apply -f` the above files on your management cluster and it should start provisioning the new cluster.
+Clusters that are provisioned by the management cluster that run your application workloads are called [Workload Clusters](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/GLOSSARY.md#workload-cluster).
+
+The `kubeconfig` file to access workload clusters should be accessible as Secrets on the management cluster. As of today, the
+secret is named `<cluster-resource-uuid>-kubeconfig` in the same namespace as the cluster resource it belongs to. For example,
+if a cluster resource in the `default` namespace has UUID `3644a115-88ae-11e9-952c-005056b08e9e`, you can fetch the kubeconfig
+by finding the secret like so:
+```
+$ kubectl -n default get secrets 3644a115-88ae-11e9-952c-005056b08e9e-kubeconfig -o yaml
+```
+
+Now that you have the `kubeconfig` for your Workload Cluster, you can start deploying your applications there.
```

Comment:
Hi @andrewsykim,

It may be valuable to actually add a Terms and Phrases section at the top of this document to define things like "Workload Cluster" there. I personally would find that valuable. Thoughts?

Reply:
I've noticed the terminology trips a lot of people up so this would be helpful.  A graphic showing the relations may also be helpful.

Reply:
The doc links out in many places to the glossary page in the cluster-api docs but might be helpful to be more explicit in the beginning that the glossary page exists. 

If we feel the glossary page there (https://cluster-api.sigs.k8s.io/glossary.html) is not enough we can add the definitions here again.